src/pages/OrderPage.java

```diff
@@ -0,0 +1,9 @@
+package pages;
+
+public class OrderPage {
+
```

Comment:
Where is order page actions/methods? 
where is the logic?

Reply:
orderId added.